lib/localisation/locales.ts

```diff
@@ -1,2 +1,8 @@
-// ! Remember to modify the middleware.ts file as you add or remove locales here
-export const SUPPORTED_LOCALES = ['en', 'es'] as const;
+export const SUPPORTED_LOCALES = ['en', 'es', 'ar'] as const;
+
+export type Locale = (typeof SUPPORTED_LOCALES)[number];
+
+export const getDirection = (locale: Locale) => {
+  const rtlLocales = ['ar'];
+  return rtlLocales.includes(locale) ? 'rtl' : 'ltr';
+};
```

Comment:
Remove, in favour of lib.

Reply:
removed [4dd0b38](https://github.com/complexdatacollective/Studio/pull/35/commits/4dd0b382229ae2443c1beef181b74c500d12cc2c)